### ISSUE TITLE
Cannot initialize UILabel node until content for Label.xml has been loaded 

### ISSUE THREAD

**Issue by micpringle** [schibsted/layout#76]
Now I have a working layout I'm attempting to break it up into templates so that they can be reused across 3 very similar views, but after recreating the same layout using those templates I've run into an issue where I come up against the red screen and the following error is displayed:

```
Cannot initialize UILabel node until content for Label.xml has been loaded.
```

Here are the XML files I'm using...

**Claims.xml**
```XML
<UIView template="Background.xml">
  <UIView template="SafeAreaContainer.xml">
    <UIView template="ContentContainer.xml">
      <UILabel
        template="Label.xml"
        font="Roboto-Regular 21.0"
        text="If you need to claim"/>
      <UILabel
        template="Label.xml"
        top="previous.bottom + 8.0"
        font="Roboto-Light 17.0"
        text="Should you need to make a claim on your policy, please contact us using one of the numbers below"/>
      <UIView
        template="Contact.xml"
        top="previous.bottom + 8.0"
        text="In the UK"
        number="08000 156 166"/>
      <UIView
        template="Contact.xml"
        top="previous.bottom + 8.0"
        text="From outside the UK"
        number="(0044) 161 866 4114"/>
      <UILabel
        template="Label.xml"
        top="previous.bottom + 8.0"
        font="Roboto-Regular 17.0"
        text="Telephone Call Charges"/>
      <UILabel
        template="Label.xml"
        top="previous.bottom + 8.0"
        font="Roboto-Light 12.0"
        text="Calls to 0800 numbers are free from landlines and mobiles. Calls to 01 and 03 numbers cost the same as a standard landline call, even when dialing from mobiles."/>
    </UIView>
  </UIView>
</UIView>
```

**Contact.xml**
```XML
<UIView
  width="100%"
  height="auto">
  <param name="text" type="String"/>
  <param name="number" type="String"/>
  <UILabel
    template="Label.xml"
    font="Roboto-Regular 21.0"
    text="{text}"/>
  <UILabel
    template="Label.xml"
    top="64 - height"
    font="Roboto-Light 22.0"
    text="{number}"/>
  <UIImageView
    width="64"
    height="64"
    right="100%"
    image="Phone"/>
</UIView>
```

**Label.xml**
```XML
<UILabel
  font="{font}"
  numberOfLines="0"
  textColor="white"
  text="{text}">
  <param name="font" type="String"/>
  <param name="text" type="String"/>
</UILabel>
```

I've narrowed the issue down to the use of the `Label.xml` template within `Contact.xml`, but **only when** `Contact.xml` **is included more than one once**.

If I comment out either instance of the `UIView`'s using `Contact.xml` in `Claims.xml`, everything works as expected. If I comment out the uses of the `Label.xml` template in `Contact.xml`, then I can include both instances of `Contact.xml` in `Claims.xml`and everything renders fine. But as soon as I try and use `Label.xml` from within `Contact.xml`, I hit the error.

(I've included the sample project I've been working with [here](https://github.com/schibsted/layout/files/1434032/AlternateLayout.zip) as my description above might not be the clearest)

Any ideas?

**Comment by nicklockwood** [comment]
Thanks for including the project, that was a big help!

Predictably, it was a caching problem. Fixed for next release.

**Comment by micpringle** [comment]
Fantastic news! Cheers once again Nick. I owe you a 🍺

**Comment by nicklockwood** [comment]
@micpringle I've pushed this and the font fixes to the develop branch if you want to test them. I'll probably cut another release tomorrow.

**Comment by micpringle** [comment]
@nicklockwood Can confirm this is now working as expected. Thanks once again for the quick turnarounds on these issues.

**Comment by nicklockwood** [comment]
Fixed in 0.5.9
